src/content/en/2020/markup.md

```diff
@@ -0,0 +1,560 @@
+---
+part_number: I
+chapter_number: 3
+title: Markup
+description: Markup chapter of the 2020 Web Almanac covering general observations, the use of elements and attributes, as well as trivia and trends.
+authors: [j9t, catalinred, iandevlin]
+reviewers: [zcorpan, matuzo, bkardell]
+analysts: [tiggerito]
+translators: []
+discuss: 2039
+results: https://docs.google.com/spreadsheets/d/1Ta7amoUeaL4pILhWzH-SCzMX9PsZeb1x_mwrX2C4eY8/
+queries: 03_Markup
+# featured_quote: @@
```

Comment:
Can you also add the author bios? Last bit of metadata I promise!

```suggestion
# j9t_bio: @@
# catalinred_bio: @@
# iandevlin_bio: @@
# featured_quote: @@
```

See here for examples from last year:

https://github.com/HTTPArchive/almanac.httparchive.org/blob/577fe1b2604336d29d0fdc720fb24c4c22b7e17b/src/content/en/2019/markup.md#L12-L20

Reply:
Added (coming up). I can’t (or don’t) test at the moment but expect this will accept Markdown, too?

Reply:
It accepts HTML but not Markdown